geomesa-tutorials-accumulo/geomesa-tutorials-accumulo-lambda-quickstart/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Apache License, Version 2.0 which
+  ~ accompanies this distribution and is available at
+  ~ http://www.opensource.org/licenses/apache2.0.php.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.geomesa.example</groupId>
+        <artifactId>geomesa-tutorials-accumulo</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>geomesa-tutorials-accumulo-lambda-quickstart</artifactId>
+    <name>GeoMesa Tutorials - Accumulo - Lambda Quick Start</name>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.apache.curator</groupId>
+                <artifactId>curator-recipes</artifactId>
+                <version>2.7.1</version>
```

Comment:
Should have a comment on why we're using a different version here.

Reply:
hmm, why are we using a different version? ;)

Reply:
seems like this is the same version in the main pom, i'll remove it

Reply:
i reorganized the poms a bit - for some reason this was set to 2.11 in the main pom. I removed that and verified all the quick starts still work. I moved the dependency management sections into each back-end module, something i had started but not completed